swift-lint-sample/AppDelegate.swift

```diff
@@ -9,9 +9,9 @@
 import UIKit
 
 @UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDelegate {
+class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDelegate  {
```

Comment:
ERROR: [Warning][opening_brace] Opening Brace Spacing
Opening braces should be preceded by a single space and on the same line as the declaration.